test-basic/documents-query.js

```diff
@@ -497,6 +502,24 @@ describe('document query', function(){
       .catch(done);
     });
 
+    it('should not throw error when queryBuilder.extract has namespace bindings', function(done){
+      db.documents.query(
+        q.where(
+            q.word('target','match')
+          ).
+        slice(0, 1, q.extract({
+          selected:'include',
+          paths:'//abc:elem',
+          namespaces: {'abc': 'http://marklogic.com/test/abc'}
+          }))
+        )
+      .result(function(response) {
+        response.length.should.equal(1);
```

Comment:
Similar to the preceding extract test, should the test verify that document.content contains the expected string for the extracted XML?

Reply:
Yes that’s right. Adding the code for content check.